src/x/clock/options.go

```diff
@@ -32,6 +38,14 @@ type options struct {
 
 // NewOptions creates new clock options.
 func NewOptions() Options {
+	if os.Getenv(panicOnDefaultClockEnvVar) == "true" {
```

Comment:
Maybe we could properly parse boolean string so that condition would still hold if env var is set to e.g. "True", etc.
Probably it would be enough to use `strconv.ParseBool`.